test/bench/granular/common.ts

```diff
@@ -71,7 +72,7 @@ export function readEnvVars(): { warmup: number; iterations: number; library: st
   const rv = {
     warmup: Number.isSafeInteger(envWarmup) && envWarmup > 0 ? envWarmup : 100_000,
     iterations: Number.isSafeInteger(envIterations) && envIterations > 0 ? envIterations : 10_000,
-    library: libraryPath ? `bson:${libraryPath}` : 'bson#main'
+    library: libraryPath ? `bson:${libraryPath}` : `bson:${process.cwd()}`
```

Comment:
from the AC:

> How do we fix the fact that the test names include the branch "main" the name needs to remain the same in order to get history.

I think this will break the perf tooling's continuity.  We can default to `cwd()`, but we still need to name anything that runs on `main` as `main`.

Reply:
This was already fixed in the last PR, if you open the trend charts you can see the history hasn't been restarted and there is no branch name in the tests